SampleApp/AppDelegate.swift

```diff
@@ -16,6 +16,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
   func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
     // Override point for customization after application launch.
+    MapzenManager.sharedManager.apiKey = "mapzen-2qQR7SX"
```

Comment:
Its great to have this centralized in a single location now. Before we release the final version of the SDK this API key should be replaced with something like `[YOUR_MAPZEN_API_KEY]` and instructions should be added to the README for setting the API key. This will help prevent accidental re-use of house demo API keys by external developers.

https://github.com/mapzen/ios/issues/145